recipe/0001-Use-userelocatableinc-together-with-useshrplib.patch

```diff
@@ -0,0 +1,27 @@
+diff --git Configure Configure
+index cef8c3c..9e96f1d 100755
+--- Configure
++++ Configure
+@@ -8712,13 +8712,6 @@ esac
+ 
+ case "$useshrplib" in
+ true)
+-	case "$userelocatableinc" in
+-	true|define)
+-		echo "Cannot build with both -Duserelocatableinc and -Duseshrplib" >&4
+-		echo "See INSTALL for an explanation why that won't work." >&4
```

Comment:
> ```diff
> -		echo "See INSTALL for an explanation why that won't work." >&4
> ```
Refers to https://github.com/Perl/perl5/blob/v5.32.0/INSTALL#L900-L902:
> because in that case perl is linked with an hard-coded
rpath that points at the libperl.so, that cannot be relocated.

Since `conda-build` makes our rpaths relative with `$ORIGIN/<relative-path>`, the "hard-coded rpath ... cannot be relocated" doesn't really apply for us.

The `sed` part of the patch below help with this. Perl's `ccdlflags` carries the accompanying `-Wl,-rpath,...`.
(Other `lddflags*`/`*dlflags` don't pass on `-Wl,-rpath,...`, which is somewhat odd, but probably expected since extension modules don't seem to have the linking library recorded, ref https://github.com/conda-forge/perl-feedstock/issues/48#issuecomment-732337864 .)